src/genegraph/source/registry/redis.clj

```diff
@@ -0,0 +1,77 @@
+(ns genegraph.source.registry.redis
+  (:require [genegraph.rocksdb :as rocksdb]
+            [io.pedestal.log :as log]
+            [taoensso.carmine :as car]))
+
+;; Carmine uses memoization to store the threadpool for each connection config.
+;; This might leave dangling threads in the pool even when the application code
+;; has finished. Should look into that.
+;; https://github.com/ptaoussanis/carmine/issues/224
+;; https://github.com/ptaoussanis/carmine/issues/266
+;; Might be able to get the handle to the threadpool by calling
+;; taoensso.carmine.connections/conn-pool.
+;; Might also be able to kick the threadpool out of hte memoize cache
+;; by sending the db spec with :mem/del as the first vararg.
+;; http://ptaoussanis.github.io/encore/taoensso.encore.html#var-memoize
+(def conn {:pool {} :spec {:uri "redis://localhost:6379/"}})
+
+(defmacro wcar* [& body] `(car/wcar conn ~@body))
+
+(wcar* (car/ping)
```

Comment:
Going to need some explanations here

Reply:
`wcar` is a function in the carmine library that handles connection pooling and command pipelining/transactions. So you can pass multiple `car/get` or `car/set` commands in a single `car/wcar` call, and they will be sent in a batch and executed in sequence and the results will all be transmitted back to client in one response. 

It implements this concept:
https://redis.io/docs/manual/pipelining/#redis-pipelining